addon/mixins/route.js

```diff
@@ -193,6 +199,7 @@ const RouteMixin = Ember.Mixin.create({
       currentPage: startingPage,
       _firstPageLoaded: false,
       _perPage: perPage,
+      _store: store,
```

Comment:
Also need to make sure that their method returns a promise as well.  [here](https://github.com/Redsandro/ember-infinity/blob/d91db9d50647e61dbfd58f2b03769f06d3273748/addon/mixins/route.js#L283)